### ISSUE TITLE
"composer thanks" returning an exception

### ISSUE THREAD

**Issue by InfRandomness** [symfony/thanks#79]
Hello, just installed this package and when I'm running the command "composer thanks" in the root of my repo, composer thanks return me : 

                       
  [ErrorException]       
  Undefined index: data  
                         
and just after : 

thanks [--dry-run]

I checked on two packages on GitHub and no stars have been given to them.

**Comment by emodric** [comment]
Happens also on every `composer update` when some packages are upgraded.

**Comment by nicolas-grekas** [comment]
what the stack trace when this happens? Can you dump the returned data structure that is missing the data key?

**Comment by emodric** [comment]
@nicolas-grekas Here's the stack trace:

```
Downloading https://api.github.com/graphql
Retrying download: 

                         
  [ErrorException]       
  Undefined index: data  
                         

Exception trace:
 () at /var/www/html/sf/vendor/symfony/thanks/src/GitHubClient.php:199
 Composer\Util\ErrorHandler::handle() at /var/www/html/sf/vendor/symfony/thanks/src/GitHubClient.php:199
 Symfony\Thanks\GitHubClient->call() at /var/www/html/sf/vendor/symfony/thanks/src/GitHubClient.php:160
 Symfony\Thanks\GitHubClient->getRepositories() at /var/www/html/sf/vendor/symfony/thanks/src/Thanks.php:84
 Symfony\Thanks\Thanks->displayReminder() at n/a:n/a
 call_user_func() at phar:///home/edi/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:176
 Composer\EventDispatcher\EventDispatcher->doDispatch() at phar:///home/edi/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:96
 Composer\EventDispatcher\EventDispatcher->dispatchScript() at phar:///home/edi/bin/composer/src/Composer/Installer.php:324
 Composer\Installer->run() at phar:///home/edi/bin/composer/src/Composer/Command/UpdateCommand.php:163
 Composer\Command\UpdateCommand->execute() at /home/edi/.config/composer/vendor/symfony/flex/src/Command/UpdateCommand.php:38
 Symfony\Flex\Command\UpdateCommand->execute() at phar:///home/edi/bin/composer/vendor/symfony/console/Command/Command.php:245
 Symfony\Component\Console\Command\Command->run() at phar:///home/edi/bin/composer/vendor/symfony/console/Application.php:835
 Symfony\Component\Console\Application->doRunCommand() at phar:///home/edi/bin/composer/vendor/symfony/console/Application.php:185
 Symfony\Component\Console\Application->doRun() at phar:///home/edi/bin/composer/src/Composer/Console/Application.php:267
 Composer\Console\Application->doRun() at phar:///home/edi/bin/composer/vendor/symfony/console/Application.php:117
 Symfony\Component\Console\Application->run() at phar:///home/edi/bin/composer/src/Composer/Console/Application.php:106
 Composer\Console\Application->run() at phar:///home/edi/bin/composer/bin/composer:61
 require() at /home/edi/bin/composer:24
```

The resulting array contains a large list of errors, all same as this one:

```
array(1) {
  ["errors"]=>
  array(542) {
    [0]=>
    array(3) {
      ["type"]=>
      string(19) "INSUFFICIENT_SCOPES"
      ["locations"]=>
      array(1) {
        [0]=>
        array(2) {
          ["line"]=>
          int(2)
          ["column"]=>
          int(77)
        }
      }
      ["message"]=>
      string(279) "Your token has not been granted the required scopes to execute this query. The 'platform' field requires one of the following scopes: ['public_repo'], but your token has only been granted the: [''] scopes. Please modify your token's scopes at: https://github.com/settings/tokens."
    }
```

**Comment by InfRandomness** [comment]
It looks like the error disappeared, I couldn't get it reproduced. So I checked the stars I gave with my github account and a whole lot appeared, so, I think it works.

**Comment by nicolas-grekas** [comment]
Should be fixed by #79

**Comment by glensc** [comment]
correct link is #80
